### ISSUE TITLE
API returning wrong payout

### ISSUE THREAD

**Issue by EvertonSpohr** [iqoptionapi/iqoptionapi#48]
The API is returning the wrong payout for digital transactions, always returning 60% and the purchase is also being made with this payout.

**Comment by SpikeBarrier** [comment]
I found this solution on another version of the API on github which I have tested and found it works:

"I found the solution, I need to use the version V2 of the function:

iq.buy_digital_spot_v2(ACTIVES,amount,action,duration)"

**Comment by EvertonSpohr** [comment]
Thanks for the solution, it worked. To get the result I use the following function, I hope it helps.

while True:
    try:
        check, win = ROBO.API.check_win_digital_v2(id)
        time.sleep(0.5)

        if check == True:
            RESULT = round(float(win), 2)           
            
            if RESULT < 0:
                print("LOSS")                
                break
            else:
                print("WIN")                 
                break

    except Exception as e:               
        
        ck_result = False
        RESULT = 0
        
        break

**Comment by SpikeBarrier** [comment]
Thanks, that's helpful!

I was actually looking for the current spot profit which used to work on get_digital_current_profit but you can replace that with                 get_digital_payout.

**Comment by trolliama** [comment]
The api has a method called get_digital_payout?

**Comment by TheHolsback** [comment]
yep, and works really good. But i'm looking for a way to get a binary Payout. Anyone knows?

**Comment by vinifr** [comment]
> yep, and works really good. But i'm looking for a way to get a binary Payout. Anyone knows?

You can get it after open an operation from json["name"] == "option-opened", json["microserviceName"] == "binary-options"
json["msg"]["turbo"]["actives"][activeStr]["option"]["profit"]["commission"]
